### ISSUE TITLE
Doesn't use port specifed in config

### ISSUE THREAD

**Issue by dsandstrom** [jc00ke/guard-puma#17]
I set the port I want to use in puma's config file to 4010, then made guard-puma load that file.  However, port 4000 is still used.  I have to set the port using guard-puma's options.

Doesn't work

``` rb
guard 'puma', config: 'config/puma/development.rb' do
...
```

Work around

``` rb
guard 'puma', config: 'config/puma/development.rb', port: 4010 do
...
```

**Comment by jc00ke** [comment]
Thanks for the report. Can you set up a barebones repo that demonstrates this? Thanks!

**Comment by dsandstrom** [comment]
Looks like this was fixed by #15, you just need to release a new gem version.

**Comment by jc00ke** [comment]
@dsandstrom heh, you're right. Sorry for the slow release, it's been a long time since I've even looked at this project.

https://rubygems.org/gems/guard-puma
